class-00/lab-a/lab/js/app.js

```diff
@@ -72,7 +72,7 @@ function handleClick(event) {
 function showList() {
   for(let i = 0; i < allProducts.length; i++) {
     const liEl = document.createElement('li');
-    liEl.textContent = allProducts[i].name + ` has ` + allProducts[i].votes + ` votes in ` + allProducts[i].views + ` views`;
+    liEl.textContent = `${allProducts[i].name} + has + ${allProducts[i].votes} + votes in + ${allProducts[i].views} + views`;
```

Comment:
`${allProducts[i].name} has ${allProducts[i].votes} votes in ${allProducts[i].views} views`